src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -356,6 +371,18 @@ export const PersonTableRow = ({
                   {t('common.save')}
                 </LoadingButton>
               </DialogActions>
+              <ConfirmDialog
+                open={openConfirmDeleteDialog}
+                title={t('basic_data.person_register.remove_employment_title')}
+                onAccept={() => {
+                  const filteredEmployments = values.employments.filter((_, index) => index !== employmentIndex);
+                  setFieldValue('employments', filteredEmployments);
+                  setEmploymentIndex(0);
+                  toggleConfirmDeleteDialog();
+                }}
+                onCancel={() => toggleConfirmDeleteDialog()}>
```

Comment:
Når det ikke skal sendes noe argument kan det forenkles til å fjerne initialisering av ny funksjon også:
```suggestion
                onCancel={toggleConfirmDeleteDialog}>
```